From 4bcc5a1e0c79a61175356bef899c36207730be6d Mon Sep 17 00:00:00 2001
From: Giovanni Bussi <giovanni.bussi@gmail.com>
Date: Mon, 13 Jun 2016 18:54:34 +0200
Subject: [PATCH] Added all_plus_docs target

useful for macports
---
 Makefile | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4c88c89..9b0e5a4 100644
--- a/Makefile
+++ b/Makefile
@@ -9,13 +9,19 @@ SUBDIRS := $(SRCDIRS) user-doc developer-doc regtest
 SUBDIRSCLEAN:=$(addsuffix .clean,$(SUBDIRS))
 
      
-.PHONY: all lib clean $(SRCDIRS) doc docclean check cppcheck distclean
+.PHONY: all lib clean $(SRCDIRS) doc docclean check cppcheck distclean all_plus_docs
 
 # if machine dependent configuration has been found:
 ifdef GCCDEP
 all:
 	$(MAKE) lib
 
+# target useful for macports
+# it builds the code then the documentation
+all_plus_docs:
+	$(MAKE) all
+	$(MAKE) docs
+
 lib:
 	$(MAKE)	-C src
 
-- 
2.8.3

